template/tmpl/method.tmpl

```diff
@@ -80,7 +85,7 @@ var self = this;
     } } ?>
 
     <?js if (data.returns && returns.length) { ?>
-    <?js= this.partial('returnsHeading.tmpl', returns) ?>
+    <h5>Returns:</h5>

```

Comment:
TODO - decide whether to keep the "new" returns Heading template- seems alot of duplication?